packages/grafana-data/src/field/overrides/processors.ts

```diff
@@ -24,6 +24,18 @@ export const numberOverrideProcessor = (
   return parseFloat(value);
 };
 
+export interface SliderFieldConfigSettings {
+  min: number;
+  max: number;
+  step?: number;
+}
+
+export const sliderOverrideProcessor = (
+  value: any,
+  _context: FieldOverrideContext,
+  _settings?: SliderFieldConfigSettings
+) => value as number | number[];
```

Comment:
Question: in what scenario could we return an array of numbers?

Reply:
@mckn it's for the range slider I assume. See my comment here: https://github.com/grafana/grafana/pull/28007#discussion_r499558978 :)

Reply:
Hi @mckn thx for you feedback. Do you have a specific component in mind (ref. feedback point 2)?

Reply:
I think a good use case for starting to use this is the [graph3 plugin](https://github.com/grafana/grafana/blob/master/public/app/plugins/panel/graph3/module.tsx) as @dprokop suggested

Reply:
Yes, I agree! Go with that one!